src/v2/Apps/Unsubscribe/Components/UnsubscribeLoggedIn.tsx

```diff
@@ -0,0 +1,96 @@
+import { Spacer, Button, Select, Banner } from "@artsy/palette"
+import React, { useRef, useState } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { useSystemContext } from "v2/Artsy"
+import { UpdateUserEmailPreferencesMutation } from "v2/Components/UserSettings/UserEmailPreferences/UserEmailPreferencesMutation"
+import { UnsubscribeLoggedIn_me } from "v2/__generated__/UnsubscribeLoggedIn_me.graphql"
+
+enum Mode {
+  Pending,
+  Loading,
+  Success,
+  Error,
+}
+
+interface UnsubscribeLoggedInProps {
+  me: UnsubscribeLoggedIn_me
+}
+
+export const UnsubscribeLoggedIn: React.FC<UnsubscribeLoggedInProps> = ({
+  me,
+}) => {
+  const { relayEnvironment } = useSystemContext()
+
+  const [emailFrequency, setEmailFrequency] = useState(
+    me.emailFrequency ?? "daily"
```

Comment:
We could possibly come back to this page and use this same fallback for signed out users but not worth holding up this PR - just something I was chatting with Jenna about.